fastestimator/dataset/data/horse2zebra.py

```diff
@@ -19,14 +19,13 @@
 
 import wget
 
-from fastestimator.dataset.unlabeled_dir_dataset import UnlabeledDirDataset
-from fastestimator.dataset.unpaired_dataset import UnpairedDataset
+from fastestimator.dataset.dir_dataset import DirDataset
 from fastestimator.util.wget_util import bar_custom, callback_progress
 
 wget.callback_progress = callback_progress
 
 
-def load_data(root_dir: Optional[str] = None) -> Tuple[UnpairedDataset, UnpairedDataset]:
+def load_data(root_dir: Optional[str] = None) -> Tuple[DirDataset]:
```

Comment:
-> Tuple[BatchDataset, BatchDataset]

(tuple typing should reflect the exact number of args in the tuple)

Reply:
fixed